src/hooks/useToggleBoolean.ts

```diff
@@ -0,0 +1,13 @@
+import { useState } from 'react'
+
+type ReturnType = [boolean, () => void]
+
+export const useToggleBoolean = (init: boolean): ReturnType => {
+  const [status, setStatus] = useState(init)
+
+  const handleToggleStatus = () => {
+    setStatus(!status)
+  }
+
+  return [status, handleToggleStatus]
+}
```

Comment:
```suggestion
export const useBoolean = (initialValue: boolean) => {
  const [value, setValue] = useState(initialValue)

  const toggle = () => {
    setValue(!value)
  }

  return { value, toggle }
}
```
리턴 타입이 복잡하지 않아서 자동으로 추론되게 하는 것도 좋을 것 같습니다!

boolean 값에 대한 toggle 제어를 원하는 훅으로 보여서 네이밍을 useBoolean으로 변경하고 toggle을 내보내는 건 어떠신가요?

Reply:
앗 요 부분 제거하는 것을 잊었네요 ! 현재 사용하지 않고 있어서 제거하도록 하겠습니다 !
좋은 의견 감사해용  ☺️

Reply:
[반영완료](https://github.com/price-offer/offer-fe/pull/203/commits/f4418950e2c876977d9d0097c520cd0746d2f0f0)